analysis/import/GitLogParser/README.md

```diff
@@ -32,35 +32,66 @@ The names of authors are saved when the --add-author flag is set.
+
+See `ccsh gitlogparser repo-scan -h` for help. Standard usage:
+
+> `ccsh gitlogparser repo-scan --repo-path <path>`
+
+With the sub command `repo-scan`, the git log and the file name list are created automatically either from the
+current working directory or from the directory at `repo-path`.
+
+The result is written as JSON to standard out or into an output file (if specified by `-o` option).
+
+If a project is piped into the GitLogParser, the results and the piped project are merged.
+The resulting project has the project name specified for the GitLogParser.
+
+#### Example using Git
+
+-   `cd <my_git_project>`
+-   `./ccsh gitlogparser repo-scan -o output.cc.json`
+-   load `output.cc.json` in visualization
+
+### Manual creation of required files | log-scan
+
+#### Creating the repository log for metric generation
 
 | SCM | Log format                   | Command for log creation                            | tracks renames | ignores deleted files | supports code churn |
 | --- | ---------------------------- | --------------------------------------------------- | -------------- | --------------------- | ------------------- |
 | git | GIT_LOG_NUMSTAT_RAW_REVERSED | `git log --numstat --raw --topo-order --reverse -m` | yes            | yes                   | yes                 |
 
 You can also use the bash script anongit which generates an anonymous git log with log format GIT_LOG for usage with CodeCharta.
 
-### Creating the git files list of the repository for metric generation
+#### Creating the git files list of the repository for metric generation
 
 > `git ls-files > file-name-list.txt`
 
 Please make sure to execute this command in the root folder of your repository.
 
-### Executing the GitLogParser
+#### Executing the GitLogParser
+
+See `ccsh gitlogparser log-scan -h` for help. Standard usage:
 
-See `ccsh -h` for help. Standard usage:
+> `ccsh gitlogparser log-scan --git-log <path> --repo-files <path>`
 
-> `ccsh gitlogparser <log_file> -n <file-name-list>`
+With the sub command `log-scan`, an existing git log and file name list are used for parsing.
 
 The result is written as JSON to standard out or into an output file (if specified by `-o` option).
 
 If a project is piped into the GitLogParser, the results and the piped project are merged.
 The resulting project has the project name specified for the GitLogParser.
 
-### Example using Git
+## Examples
+
+### Automatic usage
+
+-   `./ccsh gitlogparser repo-scan --repo-path <path_to_my_git_project> -o output.cc.json`
+-   load `output.cc.json` in visualization
+
+### Manual usage
 
 -   `cd <my_git_project>`
 -   `git log --numstat --raw --topo-order --reverse -m > git.log` (or `anongit > git.log`)
 -   `git ls-files > file-name-list.txt`
--   `./ccsh gitlogparser git.log -o output.cc.json -n file-name-list.txt`
+-   `./ccsh gitlogparser log-scan --git-log git.log --repo-files file-name-list.txt -o output.cc.json`
 -   load `output.cc.json` in visualization
```

Comment:
I like this "full example". We could move it up to the section where the manual parsing is described.